docs/user-guide/cli-installcli.md

```diff
@@ -8,8 +8,10 @@ Install Zowe&trade; CLI on your computer.
 
 Use one of the following methods to install Zowe CLI.
 
-- [Installing Zowe CLI from a local package](#installing-zowe-cli-from-a-local-package)
-- [Installing Zowe CLI from an online registry](#installing-zowe-cli-from-an-online-registry)
+- [Installing Zowe CLI](#installing-zowe-cli)
+  - [Methods to install Zowe CLI](#methods-to-install-zowe-cli)
```

Comment:
Should this list include the two headings from above?

Reply:
No, some toc-related code got deleted in the md file. Fixed.